### PR TITLE
Clarify the file handling

### PR DESCRIPTION
**Pull request by david-caro**
Some small changes to clarify the file handling and allow easier
debugging of the same (and other workflow related actions)

### CODE REVIEW

**Comment by david-caro** [comment]
@jmartinm this is what I was talking about before :)

File: inspirehep/utils/helpers.py [hunk 1]
```diff
@@ -42,12 +42,23 @@ def download_file(url, output_file=None, chunk_size=1024):
...
 
 
-def download_file_to_record(record, name, url):
-    """Download a file to specified record."""
+def download_file_to_workflow(workflow, name, url):
+    """Download a file to specified workflow.
+
+    Here are some tips for debugging (might be old when reading this, but it's
+    a good hint).
+    The workflow.files propertiy it's actually a method that returns a
```
Comment (jmartinm) [inline inspirehep/utils/helpers.py]: typo ;) `propertiy` -> `property`

**Comment by david-caro** [comment]
I'll split this into three PRs as there are three things that seem different

File: inspirehep/utils/helpers.py [hunk 2]
```diff
@@ -42,12 +42,23 @@ def download_file(url, output_file=None, chunk_size=1024):
...
 
 
-def download_file_to_record(record, name, url):
-    """Download a file to specified record."""
+def download_file_to_workflow(workflow, name, url):
+    """Download a file to specified workflow.
+
+    Here are some tips for debugging (might be old when reading this, but it's
+    a good hint).
+    The workflow.files property it's actually a method that returns a
```
Comment (kaplun) [inline inspirehep/utils/helpers.py]: typo: `it's` -> `is`

**Pull request closed without merging**